rest/model/task.go

```diff
@@ -143,6 +144,12 @@ func (at *APITask) BuildFromService(t interface{}) error {
 			Requester:         ToStringPtr(v.Requester),
 			Aborted:           v.Aborted,
 		}
+
+		if v.HostId != "" {
+			hostLink := fmt.Sprintf("%s/host/%s", evergreen.GetEnvironment().Settings().ApiUrl, v.HostId)
```

Comment:
Assuming you want to link to the host page, this should be the UI URL, not API URL